lib/__tests__/standalone-cache.test.mjs

```diff
@@ -404,7 +404,12 @@ describe('standalone cache uses cacheStrategy', () => {
 		expect(results.some((file) => isChanged(file, newFileDest))).toBe(true);
 
 		// No content change, but file metadata id changed
-		await utimes(newFileDest, new Date(), new Date());
+		// Note: needs a time offset for fast test machines
+		const time = new Date();
+
+		time.setSeconds(time.getSeconds() + 10);
+		await utimes(newFileDest, time, time);
```

Comment:
This test was failing on my machine, also on main.

I am assuming that the precision of timestamps on mac is not very high and that modern macs (M1 or later) are just very fast.

This combo means there just isn't enough time between creating the file, running Stylelint once and calling `utimes()`.

Artificially increasing the time delta makes this test pass consistently.

Reply:
That makes sense. 👍🏼 

In addition, we could simplify the code (`Date.now()` returns milliseconds) since [`fs.utimes()`](https://nodejs.org/api/fs.html#fsutimespath-atime-mtime-callback) can also receive numbers. Like so:

```js
await utimes(newFileDest, Date.now(), Date.now() + 1000);
```

Also, it seems possible to change also in another case: 👇🏼 

https://github.com/stylelint/stylelint/blob/0500ce6e40a8f9a978e3fe0ae360626bdb13d930/lib/__tests__/standalone-cache.test.mjs#L421-L422

Reply:
> I've also seen periodic failures of this test in CI.

Thanks for the fix.

Reply:
`fs.utimes` [expects time in seconds](https://github.com/nodejs/node/issues/33227), i.e. `Date.now() / 1000`.

I don't think `Date.now() / 1000 + 1` is the most readable, but this is probably fine as it is only in a test and it is accompanied by a comment.